ui/app/templates/components/form-field.hbs

```diff
@@ -225,4 +179,4 @@
     value=(if (get model valuePath) (stringify (get model valuePath)) emptyData)
     valueUpdated=(action "codemirrorUpdated" attr.name false)
   }}
-{{/if}}
+{{/if}}
```

Comment:
Are there any changes in this file other than whitespace? I think the formatting before was easier to read fwiw - if it's just whitespace changes, could we back these out?